cmd/gh/main.go

```diff
@@ -140,7 +140,7 @@ func main() {
 		fmt.Fprintln(stderr, cs.Bold("Welcome to GitHub CLI!"))
 		fmt.Fprintln(stderr)
 		fmt.Fprintln(stderr, "To authenticate, please run `gh auth login`.")
-		fmt.Fprintln(stderr, "You can also set the one of the auth token environment variables, if preferred.")
+		fmt.Fprintln(stderr, "You can also set one of the auth token environment variables, if preferred.")
```

Comment:
I've missed this in my review of https://github.com/cli/cli/pull/2388, but I think that this line isn't very useful anymore. It used to give a more specific advice to set GITHUB_TOKEN.

@samcoe: do you agree that we should maybe take the whole line out? If users want to learn about our environment variable support, they can read so in our docs.

Reply:
Pushed a fixup commit that deletes the line. Please confirm, then I'll squash the commits.

Reply:
Yup, I agree, this line is not very useful anymore. I am on board with removing it completely.

Reply:
done :)